src/perf/.editorconfig

```diff
@@ -0,0 +1,188 @@
+# editorconfig.org
```

Comment:
This file should be at the top level probably, in case we have C# code outside of perf.